### PR TITLE
`Flyout`: Reduce gap between edge of screen in mobile view (HDS-3003)

### PR DESCRIPTION
**Pull request by KristinLBradley**
### :pushpin: Summary

If merged, this PR will update the Flyout to reduce the gap between the edge of the screen from 40px to 24px in mobile (medium) view and update the related documentation.

**Web preview:** https://hds-website-git-hds-3003-update-flyout-for-mobile-hashicorp.vercel.app/components/flyout#positioning-and-responsive-sizing

### :camera_flash: Screenshots

**BEFORE:**
image

**AFTER:**
image

**DOCUMENTATION:**
(_Updated text is highlighted_)
image

### :link: External links

Jira ticket: [HDS-3003](https://hashicorp.atlassian.net/browse/HDS-3003)

***

### 👀 Component checklist

- [x] Percy was checked for any visual regression
- [x] A changelog entry was added via [Changesets](https://github.com/changesets/changesets) if needed (see [templates here](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#templates-for-npm-packages))

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-3003]: https://hashicorp.atlassian.net/browse/HDS-3003?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by KristinLBradley** [comment]
@heather for the documentation, I updated the information regarding the gap size in smaller viewports. We could update the accompanying screen shot image to match (and include the overlapped logo visible in the SideNav).

image

File: packages/components/src/styles/components/flyout.scss [hunk 1]
```diff
@@ -7,12 +7,14 @@
 // FLYOUT
 //
 
+$hds-flyout-size-medium-gap: calc( var(--hds-app-sidenav-width-minimized) / 2 );
```
[the remark this answers is not in the document]
Comment (KristinLBradley) [reply packages/components/src/styles/components/flyout.scss:10]: Good idea. I will make that change

**Comment by heatherlarsen** [comment]
> We could update the accompanying screen shot image

@KristinLBradley I updated the image, but I don't see the change in the preview. With the repo changes recently, maybe I'm missing something? Is there an additional step I need to do now?

**Comment by KristinLBradley** [comment]
> > We could update the accompanying screen shot image
> 
> @KristinLBradley I updated the image, but I don't see the change in the preview. With the repo changes recently, maybe I'm missing something? Is there an additional step I need to do now?

I will take a look. Maybe it was just taking time to build though

**Review by heatherlarsen — APPROVED** [review]
Looks great 🙂

**Pull request merged**